src/all_shaders.zig

```diff
@@ -1,7 +1,7 @@
 const std = @import("std");
 const os = std.os;
 const panic = std.debug.panic;
-const c = @import("c.zig");
+const c = @import("c_with_glfw.zig");
```

Comment:
Instead of this, I suggest to break the dependency on math.h, stdlib.h, and stdio.h, by using the zig standard library instead. Then if you do not use GLFW (or those other libs) it will not be a dependency.

Reply:
Ok I will proceed with that.